src/js/components/Nav/Nav.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { text } from "@storybook/addon-knobs/react";
+import Nav from "./Nav";
+const stories = storiesOf("Nav", module);
+
+stories.add("Default", () => (
+  <div
+    className="u-flex"
+    style={{ backgroundColor: "#FFEFA5", minHeight: "800px" }}
+  >
+    <Nav
+      mods="u-size2of12"
```

Comment:
I think fine to use mods for size of menu as long as it handles not having a mod for size.

Reply:
If you do omit the mods, it is as big as it's parent container, which should work out fine.